bin/pycbc_split_inspinj

```diff
@@ -2,28 +2,44 @@
 
 # Copyright (C) 2014 Andrew Lundgren
```

Comment:
@andrew-lundgren , this is your code, so I give you an opportunity to look over this patch to make pycbc_split_inspinj usable in our workflows. GRB generally splits injection files up for analyses, so this is used there via the same workflow tools as for splitbank